Dockerfile

```diff
@@ -2,17 +2,23 @@ FROM python:3.11-slim
 
 WORKDIR /app
 COPY ./requirements.txt /app
+
```

Comment:
I would add another copy to copy the distance files here as a "dependency" that you build with the rest of the Py requirements. then you don't have the problem with the layer